src/GlobusSupport.cc

```diff
@@ -315,7 +315,12 @@ class CertStore {
                                   globus_gsi_callback_create_proxy_callback);
     X509_STORE_set_depth(m_cert_store, GLOBUS_GSI_CALLBACK_VERIFY_DEPTH);
     X509_STORE_set_flags(m_cert_store, X509_V_FLAG_ALLOW_PROXY_CERTS);
-    m_cert_store->check_issued = globus_gsi_callback_check_issued;
+
+    #if OPENSSL_VERSION_NUMBER < 0x1110000
```

Comment:
I think you want this:
```suggestion
    #if OPENSSL_VERSION_NUMBER < 0x010101000
```
if you want 1.1.1 ; `0x1110000` is 0.17.16 if I'm reading the numbers correctly.

Reply:
Ahh you are right